slides/lecture-2/index.html

```diff
@@ -1414,7 +1414,7 @@ <h2>Base64</h2>
       representation.
     </li>
   </ul>
-  <img src="./img/base64.png" />
+  <img src="./img/base64.png" height="400vh"/>
```

Comment:
I think this should go into style attribute just like the above case.

Reply:
Sorry, I will fix that!!